env/work/srw_python/srwlib.py

```diff
@@ -15,6 +15,10 @@
 import os
 import traceback
 import uti_math
+import math
```

Comment:
I've added 
from math import *
earlier before this (to allow for calling basic math functions directly, without "math."); so this may not be needed.